src/layouting/layouting.jl

```diff
@@ -216,10 +228,10 @@ function glyph_collection(str::AbstractString, font_per_char, fontscale_px, hali
         reduce(vcat, charorigins),
         [x.extent for x in charinfos],
         [Vec2f(x.scale) for x in charinfos],
-        [rotation for x in charinfos],
-        [color for x in charinfos],
-        [strokecolor for x in charinfos],
-        [strokewidth for x in charinfos],
+        per_char_info(rotation, charinfos),
```

Comment:
hm neither what I had before nor this seems ideal. there is already a function called `attribute_per_char` at the top, this is currently used for font and textsize, but similar logic could apply to color. anyway, it feels weird to do one expansion before the glyph_collection function and one here.

Reply:
Ah, I somehow didnt find that function... So I guess I should just use that?

Reply:
Ok, this uses `attribute_per_char` now!